app/javascript/listings/components/AllListings.jsx

```diff
@@ -13,10 +13,11 @@ export const AllListings = ({
   loadNextPage,
 }) => {
   return (
-    <main class="crayons-layout__content" id="main-content">
```

Comment:
Just double checking that we meant to remove the id here as well?

Reply:
Yup! I've added it with the other main in `app/views/listings/index.html.erb` - this makes sure our skip link takes the user to the right `main` section :)

Reply:
thanks for the clear explanation!